app/src/main/java/protect/card_locker/LoyaltyCardViewActivity.java

```diff
@@ -544,8 +556,9 @@ private void showBalanceUpdateDialog() {
                 ViewGroup.LayoutParams.MATCH_PARENT,
                 ViewGroup.LayoutParams.WRAP_CONTENT
         );
-        params.leftMargin = 60;
-        params.rightMargin = 60;
+        int contentPadding = getResources().getDimensionPixelSize(R.dimen.alert_dialog_content_padding);
+        params.leftMargin = contentPadding;
+        params.rightMargin = contentPadding;
```

Comment:
I agree. I look into it

Reply:
`@dimen/mtrl_alert_dialog_picker_background_inset` should be the same value as the default one from material dialog

Reply:
Looks like the EditText has an extra padding. 

![image](https://user-images.githubusercontent.com/5681688/198833095-43bc88fc-f80e-4e36-80c9-1ffb5b3c40a7.png)

removing it, then it will not be align with the input line.
![image](https://user-images.githubusercontent.com/5681688/198833161-c02e4aea-3657-4b18-a7e9-8d0d16b5e744.png)

magic spacing to 2dp of the input looks ok. 
![image](https://user-images.githubusercontent.com/5681688/198833186-be4e7f48-3fd9-49a1-9b8c-181a868eecc5.png)

100 % alignment will only be possible, with changing the background drawable. I think this is to much.
Especially the new material components making it unnecessary